src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash/fp';
```

Comment:
We could use the platform data utils here instead, as is done in the config.
```
import dataUtils from '../../../../platform/utilities/data/index';
 const { get, omit, set } = dataUtils;
```

Reply:
Sure. I swear I've seen `_.get` used a lot in recent PRs. Should we really be using our own `get` method moving forward?

Reply:
Yeah, it definitely depends on the situation. I believe lodash works for both objects and arrays, whereas the platform utils may not. The signature is also different. The platform data utils are intended to help keep bundle sizes small, when possible. If we have ideas for updating/improving the utils so that they are more useful, we can set up some tickets for those, and use lodash for the time being.